src/documentation/pages/components/drawer/_properties.vue

```diff
@@ -0,0 +1,382 @@
+          tbody
+            tr
+              td
+                div
+                  code chiDrawerHidden
+              td
+                div Drawer has been hidden to the user. The animation has completed.
+              td
+                div
+                  code CustomEvent&lt;string&gt;
+            tr
+              td
+                div
+                  code chiDrawerHide
+              td
+                div Drawer hide method has executed, but the closing animation has not started yet
+              td
+                div
+                  code CustomEvent&lt;string&gt;
+            tr
+              td
+                div
+                  code chiDrawerShow
+              td
+                div Drawer show method has executed, but the showing animation has not started yet
+              td
+                div
+                  code CustomEvent&lt;string&gt;
+            tr
+              td
+                div
+                  code chiDrawerShown
+              td
+                div
+                  | Drawer has been shown to the user and is fully visible. The animation has completed.
+              td
+                div
+                  code CustomEvent&lt;string&gt;
+            tr
+              td
+                div
+                  code chiDrawerClickOutside
+              td
+                div
+                  | Mutating a prop locally is an anti-pattern in Vue, so the user can use this event to decide whether or not the drawer needs to be hidden.
+              td
+                div
+                  code CustomEvent&lt;string&gt;
+
+    h3 Methods 
+    section.chi-table.chi-table__options.-bordered.-my--3
+      div(style='overflow-x:auto;') 
+        table.-text(cellpadding='0' cellspacing='0')
+          thead
+            tr
+              th
+                div Method
+              th
+                div Description
+              th
+                div Returns
+              th
+                div Parameters
+          tbody
+            tr
+              td
+                div
+                  code hide() =&gt; Promise&lt;void&gt;
+              td
+                div Hides the drawer
+              td
+                div
+                  p.-text Type: Promise&lt;void&gt;
+              td
+                .-text--center -
+            tr
+              td
+                div
+                  code show() =&gt; Promise&lt;void&gt;
+              td
+                div Shows the drawer.
+              td
+                div
+                  p.-text Type: Promise&lt;void&gt;
+              td
+                .-text--center -
+            tr
+              td
+                div
+                  code toggle() =&gt; Promise&lt;void&gt;
+              td
+                div Toggles active state (show/hide)
+              td
+                div
+                  p.-text Type: Promise&lt;void&gt;
+              td
+                .-text--center -
+
```

Comment:
@josecarlosgonzalezv I think we can get the methods and events from properties generator itself. I can see all the methods and events of chi-drawer documented in docs.json.